showcase-docs/dynamic-plugins.md

```diff
@@ -505,6 +506,62 @@ Each plugin can expose multiple routes and each route is required to define its
 - `menuItem` - This property allows users to extend the main sidebar navigation and point to their new route. It accepts `text` and `icon` properties. `icon` refers to a Backstage system icon name. See [Backstage system icons](https://backstage.io/docs/getting-started/app-custom-theme/#icons) for the list of default icons and [Extending Icons Library](#extend-internal-library-of-available-icons) to extend this with dynamic plugins.
 - `config.props` - Optional. Additionally you can pass React props to the component.
 
+#### Menu items
+
+Order and parent-children relationship of plugin menu items which are in main sidebar navigation can be customized with menu items configuration:
+
+```yaml
+# app-config.yaml
+dynamicPlugins:
+  frontend:
+    <package_name>: # same as `scalprum.name` key in plugin's `package.json`
+      dynamicRoutes: # exposes full routes
+        - path: /my-plugin # unique path in the app, can override `/`
+          module: CustomModule # optional, same as key in `scalprum.exposedModules` key in plugin's `package.json`
+          importName: FooPluginPage # optional, actual component name that should be rendered
+          menuItem: # optional, allows you to populate main sidebar navigation
+            icon: fooIcon # Backstage system icon
+            text: Foo Plugin Page # menu item text
+          config:
+            props: ... # optional, React props to pass to the component
+      menuItems: # optional, allows you to configure plugin menu items in the main sidebar navigation
+        <menu_item_name>: # unique name in the plugin menu items list
+          icon: fooIcon # optional, same as `dynamicRoute.menuItem.icon`
+          title: Foo Plugin Page # optional, same as `dynamicRoute.menuItem.text`
+          priority: 10 # optional, defines the order of menu items in the sidebar
+          parent: favirotes # optional, defines parent-child relationships for nested menu items
```

Comment:
Typo here, `favorites`

Reply:
Also, what does the string identifier refer to for "Parent" would it be another entry in this map, could it also be a reference to a menu item from another plugin?